ParabolInc/parabol
# feat(teams-limit): enable usage stats when teams limit exceeded

Partially solves: #7383

**How to test:**

- Have a free tier org with a couple of billing leaders
- Org should have an `activeDomain` set (set manually in db or create an org from a user with custom email domain)
- Add org feature flag (see: https://github.com/ParabolInc/parabol/pull/7526#issue-1477287954)
- Adjust numbers [here](https://github.com/ParabolInc/parabol/pull/7517/files#diff-0ebb6687f73dcbeef95db77a23ed552ac97457d45368062605a87c17330b9f9dR6-R9) for easier test
- Run a meeting on that free tier org
- If the limit exceeded, usage stats enabled for all billing leaders, and notification created for all billing leaders (see screenshot):

<img width="588" alt="Screenshot 2022-12-13 at 19 34 17" src="https://user-images.githubusercontent.com/466991/207384068-bd2da69d-4675-4fe3-a725-5f130846ae1f.png">

Snackbar is not included in this PR.
